### PR TITLE
fix: Pressing back from recent submissions will takes you to home page…

### PR DESCRIPTION
**Pull request by Piyush789**
Fixes #22 

**Changes:**

[Add here what changes were made in this issue and if possible provide links.]

Please make sure these boxes are checked before submitting your pull request - thank you!

- [x] Run unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into a single commit by squashing them.

Please add screenshots/GIFs here if there are any changes to the UI below.

### CODE REVIEW

**Comment by immadisairaj** [comment]
@Piyush789 can you please fill the PR template. and the issue which this fixes is not #22 but rather it is #9. So, can you change the fixes and fill the `Changes:` section.

File: app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java [hunk 1]
```diff
@@ -122,6 +133,8 @@ public void onFailure(Call<Submission> call, Throwable t) {
         });
     }
 
+
+
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java:146]: Can you please remove these two new lines..

**Review by immadisairaj — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java (hunk 1 above)
Comment (Kshitij09) [reply app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java:146]: Is this fixed?

Comment (immadisairaj) [reply app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java:146]: @Piyush789 still has not yet responded to this

Comment (Kshitij09) [reply app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java:146]: should I?

Comment (immadisairaj) [reply app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java:146]: Yeah I am closing this PR. You can work.

**Pull request closed without merging**